lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -19,7 +19,7 @@ Paste==2.0.2
 PasteDeploy==1.5.2
 docutils==0.12
 wchartype==0.1
-repoze.lru==0.6
+repoze.lru==0.7
```

Comment:
By looking at py27-unit logs on Travis, that this is built from source.

Reply:
Good catch - I wonder why that is. Works for me when I force it:

```
~/w/galaxy ❯❯❯ wget https://wheels.galaxyproject.org/packages/repoze.lru-0.7-py2-none-any.whl   collection_state ✭ ✱ ◼
--2017-10-27 12:38:09--  https://wheels.galaxyproject.org/packages/repoze.lru-0.7-py2-none-any.whl
Resolving wheels.galaxyproject.org... 54.172.110.33
Connecting to wheels.galaxyproject.org|54.172.110.33|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 10906 (11K) [application/octet-stream]
Saving to: ‘repoze.lru-0.7-py2-none-any.whl’

repoze.lru-0.7-py2-none-any.w 100%[================================================>]  10.65K  --.-KB/s    in 0s

2017-10-27 12:38:10 (146 MB/s) - ‘repoze.lru-0.7-py2-none-any.whl’ saved [10906/10906]

~/w/galaxy ❯❯❯ . .venv/bin/activate                                                             collection_state ✭ ✱ ◼
(.venv) ~/w/galaxy ❯❯❯ pip install repoze.lru-0.7-py2-none-any.whl                              collection_state ✭ ✱ ◼
Processing ./repoze.lru-0.7-py2-none-any.whl
Installing collected packages: repoze.lru
  Found existing installation: repoze.lru 0.6
    Uninstalling repoze.lru-0.6:
      Successfully uninstalled repoze.lru-0.6
Successfully installed repoze.lru-0.7
```

I'm working another update that includes Babel and where I add that missing certifi - I'll see if it happens locally...